docs/pages/project/contributing/contributing-docs.md

```diff
@@ -165,7 +165,7 @@ Restart your WSL VM before moving forward.
 If you've Docker and `make` installed in your system, then you can serve the site locally
 
 ```
-$ make docker-docs
+$ make docker
```

Comment:
Why this change?? I dont think we have a `make docker` command, we have a `make docs` command.

Reply:
Hello @warunicorn19 , In the meshery Contribution guide for docs. We have steps to follow using Docker as follows,

<img width="978" alt="Screenshot 2022-04-17 at 11 07 24 AM" src="https://user-images.githubusercontent.com/52213014/163702080-a5161938-06ca-4765-947e-6da880eb32c5.png">

But when I tried to serve used the command `make docker-docs`, It says "No target defined". So I checked the [Makefile in Docs folder](https://github.com/meshery/meshery/blob/master/docs/Makefile) and [Make file in Root Folder](https://github.com/meshery/meshery/blob/master/Makefile). 
<img width="1289" alt="Screenshot 2022-04-17 at 11 05 29 AM" src="https://user-images.githubusercontent.com/52213014/163702173-2f607eb7-ea90-4dd2-bc12-aaea99bebe2b.png">
<img width="1264" alt="Screenshot 2022-04-17 at 11 05 12 AM" src="https://user-images.githubusercontent.com/52213014/163702175-a066bc23-3209-4ab3-a92c-ba29a61e62d4.png">

But `docker-docs` wasn't defined at either locations. So I made a commit in this file.

Reply:
@karnatisrinivas would you please revert this `docker-docs` change?

Reply:
Reverted the commit made to contributing-docs.md